tests/test_benchmark_KPI.py

```diff
@@ -71,34 +70,115 @@
     TOTAL_RENEWABLE_ENERGY_USE,
     TOTAL_NON_RENEWABLE_GENERATION_IN_LES,
     TOTAL_RENEWABLE_GENERATION_IN_LES,
+    ENERGY_CONSUMPTION,
+    ENERGY_CONVERSION,
+    ENERGY_PRODUCTION,
+    ENERGY_STORAGE,
+    KPI,
+    KPI_SCALARS_DICT,
+    FLOW,
+    ATTRIBUTED_COSTS,
 )
 
 TEST_INPUT_PATH = os.path.join(TEST_REPO_PATH, "benchmark_test_inputs")
 TEST_OUTPUT_PATH = os.path.join(TEST_REPO_PATH, "benchmark_test_outputs")
 
-dict_economic = {
+DICT_ECONOMIC = {
     CURR: "Euro",
     DISCOUNTFACTOR: {VALUE: 0.08},
     PROJECT_DURATION: {VALUE: 20},
 }
 
-dict_economic.update(
+DICT_ECONOMIC.update(
     {
         ANNUITY_FACTOR: {
             VALUE: C2.annuity_factor(
-                project_life=dict_economic[PROJECT_DURATION][VALUE],
-                discount_factor=dict_economic[DISCOUNTFACTOR][VALUE],
+                project_life=DICT_ECONOMIC[PROJECT_DURATION][VALUE],
+                discount_factor=DICT_ECONOMIC[DISCOUNTFACTOR][VALUE],
             )
         },
         CRF: {
             VALUE: C2.crf(
-                project_life=dict_economic[PROJECT_DURATION][VALUE],
-                discount_factor=dict_economic[DISCOUNTFACTOR][VALUE],
+                project_life=DICT_ECONOMIC[PROJECT_DURATION][VALUE],
+                discount_factor=DICT_ECONOMIC[DISCOUNTFACTOR][VALUE],
             )
         },
     }
 )
 
+USE_CASE = "Economic_KPI_C2_E2"
+
+
+def process_expected_values():
```

Comment:
I couldn't find where you use this function. Is is used at all?

Reply:
No, this is not used! It is something that must be used when you want to change something in the original input values of the simulation (eg. adapt dispatch/om costs, discount value), but you do not have to execute it every time. Somehow PF said that this was the way to go, as we should not calculate the expected values at the same time at which we also do the assertions...

Reply:
ah okay, I see!